ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -155,6 +155,7 @@ const styles = reactStyles({
 const defaultMachineName = 'n1-standard-4';
 const defaultMachineType: Machine = findMachineByName(defaultMachineName);
 const defaultDiskSize = 100;
+const defaultPdSize = 100;
```

Comment:
I don't think we need the new const here. A PD is still a disk, should be fine to just use `defaultDiskSize`